frontend/appflowy_flutter/lib/plugins/ai_chat/presentation/chat_input/select_sources_bottom_sheet.dart

```diff
@@ -204,34 +204,27 @@ class _MobileSelectSourcesSheetBodyState
         ),
         BlocBuilder<ChatSettingsCubit, ChatSettingsState>(
           builder: (context, state) {
+            final sources = state.visibleSources
+                .where((e) => e.ignoreStatus != IgnoreViewType.hide)
+                .toList();
```

Comment:
By the way, `toList` is somewhat redundant here, I guess it's quite common to use `toList` when you have an `Iterable`, but actually it's not always neccessary.

Here you can remove `.toList`, and use `sources.elementAt(index)` instead.

It's just an observation, don't have to change it.

Reply:
Oh nice! I was trying to use `[index]` which doesn't work with iterators